presto-memsql/src/main/java/io/prestosql/plugin/memsql/MemSqlClient.java

```diff
@@ -47,7 +47,7 @@ public MemSqlClient(BaseJdbcConfig config, ConnectionFactory connectionFactory)
             while (resultSet.next()) {
                 String schemaName = resultSet.getString("TABLE_CAT");
                 // skip internal schemas
-                if (!schemaName.equalsIgnoreCase("information_schema") && !schemaName.equalsIgnoreCase("memsql")) {
+                if (filterSchema(schemaName)) {
```

Comment:
Do we still need `listSchemas` method in `MemSqlClient`?

Reply:
Yes, MemSQL also has diff as MySQL (`TABLE_CAT` vs `TABLE_SCHEM`)